Parse/TPTP.cpp

```diff
@@ -2893,13 +2893,13 @@ void TPTP::term()
           );
           break;
         case T_INT:
-          number = addIntegerConstant(tok.content,_isFof);
+          number = addNumeralConstant<IntegerConstantType>(tok.content,_isFof);
```

Comment:
Now that I think about it we just shouldn't parse FOF numerals at all - so we can remove the whole "default sort" code path for numeral constants. But I'll do that in a separate PR.